packages/subproviders/src/subproviders/injected_web3.ts

```diff
@@ -1,17 +1,16 @@
 import * as _ from 'lodash';
 import Web3 = require('web3');
-import Web3ProviderEngine = require('web3-provider-engine');
 
 /*
  * This class implements the web3-provider-engine subprovider interface and forwards
  * requests involving user accounts (getAccounts, sendTransaction, etc...) to the injected
- * web3 instance in their browser.
+ * MetamaskInpageProvider instance in their browser.
```

Comment:
Let's just say the `injected provider instance`, and not mention Metamask specifically. It could be any provider.